### DISCUSSION TITLE
Feature request: being able to use time things only for selected files

### DISCUSSION THREAD

**Discussion by luisKisters** [DynamicPlayerSector/timethings#10] [Ideas]
# First up: I understand if you're busy and I'm greatful for the plugin
Hi. So firstly I wanted to say that you're most likely very busy, and that I know that there are many other feature requests and other github issues in general, so I completely understand if you don't got time or the feature request will take a long time to fullfil. Also I like the plugin a lot and I'm **really** grateful you made it in the first place!

# Feature request: being able to use time things only for selected files
Soooo. Even though I like your plugin, I think that I won't use it for all my files. But for some files, especially journaling entries, projects, plans, daily notes and some other stuff, it would be **extremly** useful to have the time counter thing. So that's the reason I made the request. Maybe something like a command like...
-  *`"Time Things: Start tracking time in current file"`* 
-  (maybe even if there's a feature for it in the future): *`"Time Things: Pause tracking time in current file"`* 
- *`"Time Things: Remove time tracking from current file"`* 
...would be pretty cool. And just generally more commands. 

# conclusion
So yeah, as I said, I understand if you're busy or/and if you don't have time for this or if it'll take a long time. I'm grateful that you made the plugin and I think it's pretty useful and I'll most likely use it either way because it's pretty useful and valuable to me.

**Comment by plasmabit** [comment]
Great idea!
You are right, I was pretty busy recently, that fact saddens me a little bit, because I remember what it is like being on the other side and waiting for the developer to update/fix a plugin.
I really like the ideas I got from all the people, and I might start implementing them soon. There is also a bug I need to fix, which involves arrow keys. Also I might start tackling your request in particular first by just allowing to have a property which tells the plugin if a note should be tracked.
